### DISCUSSION TITLE
Can we use horizon-ui With OctoBot

### DISCUSSION THREAD

**Discussion by AtlasAlpha** [Drakkar-Software/OctoBot#1963] [Q&A] [answered]
Hi everyone! Great project! Bean playing with OctoBot and I love it! I would not mind a design upgrade thou. 

My coding skills are minuscule so I cannot figure out if there is a straight forward way of using Horizon UI (https://github.com/Drakkar-Software/horizon-ui-chakra) with OctoBot.
Could someone clarify this for me/us?

With best regards,
Atlas

**Comment by Herklos** [comment, marked as answer]
Hi @AtlasAlpha, https://github.com/Drakkar-Software/horizon-ui-chakra repo was created to do tests with horizon-ui. It is not yet possible to use it with OctoBot.
